src/modules/commander/Commander.cpp

```diff
@@ -590,7 +590,7 @@ Commander::Commander() :
 	_failure_detector(this)
 {
 	_auto_disarm_landed.set_hysteresis_time_from(false, _param_com_disarm_preflight.get() * 1_s);
-	_auto_disarm_killed.set_hysteresis_time_from(false, 5_s);
+	_auto_disarm_killed.set_hysteresis_time_from(false, _param_com_kill_disarm.get() * 1_s);
```

Comment:
Set/update this in parameter update. The auto disarm landed hysteresis gets away without it because it updates later based on `have_taken_off_since_arming`.

Reply:
Here?
https://github.com/PX4/Firmware/blob/e1b77a826707a269e85d1f792aab109df37a03c3/src/modules/commander/Commander.cpp#L1518

Reply:
Yes

Reply:
It seems a little weird having some parameters defined in the header file with `DEFINE_PARAMETERS` and using `.get()`versus others being defined in the body of `run()` using `param_find(<NAME>)`

I understand the usage seems to be the former for static parameters and the latter for dynamic, but I feel like we could just standardize around defining all parameters with `DEFINE_PARAMETERS` and then using the `param.get()` when we need to update dynamically. Thoughts?

Reply:
Yes nearly all should move over to `DEFINE_PARAMETERS`. The exception would be things like commander accessing a vehicle specific parameter.

Reply:
@dagar updated. Please review. Thanks!